### ISSUE TITLE
Exercise: Gradient descent

### ISSUE THREAD

**Issue by longqua69** [longqua69/practice-deep-learning#1]
Consider updating the notebook by creating functions from scratch without using TensorFlow API

- Loss function
- Sigmoid function
- Gradient descent function
- ....
